inference-engine/src/mkldnn_plugin/nodes/cum_sum.cpp

```diff
@@ -57,15 +58,15 @@ class CumSumImpl: public ExtLayerBase {
             if (dataShape != layer->outData[0]->getTensorDesc().getDims())
                 THROW_IE_EXCEPTION << "CumSum layer with name '" << layerName << "' has different 'data' input and output dimensions";
 
-            shape5d = get5dShape(dataShape);
+            shape = dataShape;
 
             LayerConfig config;
             for (size_t i = 0; i < layer->insData.size(); i++) {
                 DataConfig inConfig;
                 inConfig.inPlace = -1;
                 inConfig.constant = false;
 
-                Precision inPrecision = layer->insData[i].lock()->getTensorDesc().getPrecision();
+                Precision inPrecision = i == 1 ? Precision(Precision::I32) : layer->insData[i].lock()->getTensorDesc().getPrecision();
```

Comment:
minor comment
not necessary explicitly call constructor for ```Precision```
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/non_max_suppression.cpp#L122